src/main/resources/org/springframework/integration/aws/config/xml/spring-integration-aws-1.0.xsd

```diff
@@ -61,165 +75,175 @@
 		</xsd:attribute>
 	</xsd:attributeGroup>
 
-	<xsd:element name="s3-outbound-channel-adapter">
-		<xsd:complexType>
+	<xsd:complexType name="outboundS3BaseType">
+		<xsd:choice minOccurs="0" maxOccurs="2">
+			<xsd:element ref="integration:poller" minOccurs="0" maxOccurs="1"/>
+			<xsd:element name="request-handler-advice-chain" type="integration:handlerAdviceChainType" minOccurs="0" maxOccurs="1" />
```

Comment:
This is good simple fix, but even right now we have a workaround:
````xml
<int:outbound-channel-adapter channel="input">
	<bean class="org.springframework.integration.aws.s3.AmazonS3MessageHandler">
		<constructor-arg ref="awsCredentials"/>
		<constructor-arg ref="s3Operations"/>
	</bean>
	<int:request-handler-advice-chain>
		<int:retry-advice/>
	</int:request-handler-advice-chain>
</int:outbound-channel-adapter>
````

Having that I'm not sure that we need to even backport it to the {{0.6.x}}...